ultraplot/axes/geo.py

```diff
@@ -5,6 +5,18 @@
 import copy
 import inspect
 
+# Surpress warnings from cartopy when downloading data inside docs env
+import warnings
+
+try:
+    from cartopy.io import DownloadWarning
+
+    warnings.filterwarnings("ignore", category=DownloadWarning)
+except ImportError:
+    # In case cartopy isn't installed yet when conf.py is executed
+    pass
+
+
```

Comment:
I don't like ignoring warnings here by default for users. We should move this back to the docs build itself.

Reply:
So the conf did not remove the warnings from the test, do you know why?

Reply:
I do not know why. Can you point me to the code that makes the plot? I wonder if we need to embed the warnings filter inside the doc strings maybe?